lms/templates/admin/application_instance/show.html.jinja2

```diff
@@ -63,7 +72,7 @@
                         <input type="hidden" name="csrf_token" value="{{ get_csrf_token() }}">
                         <fieldset class="box">
                             {{ macros.form_text_field(request, "Name", "name", field_value=instance.name) }}
-                            {{ macros.settings_textarea(instance, "Notes", "hypothesis", "notes") }}
+                            {{ macros.settings_textarea(instance, "Notes", fields[instance.settings.Settings.HYPOTHESIS_NOTES]) }}
```

Comment:
Accessing these here is also a bit awkward

Reply:
Maybe expose the `Settings` class as a variable within this template?

Reply:
Done that.